### PR TITLE
Feb Gems draft for review

### PR DESCRIPTION
**Pull request by elleobrien**
Hey @shcheklein, @jendefig and I drafted up the Gems for this month. Would you review? We'd like to have it out by Friday since that's the last work day in February.

Questions: 

1. we have a question about dependencies when installing DVC as a Python library with `conda`. Is it correct that you need to add options for remote storage (i.e., `pip install dvc[s3]`) _only_ for pip? From the docs, it looks this way, but I wanted to confirm. 

2. Can we add _both_ Jeny and I as authors on a blog post?

Also, @jendefig, we'll need a new header image for this month. Can you do this? I can share with you the template i use!

### CODE REVIEW

**Review by jendefig — CHANGES_REQUESTED** [review]
@shcheklein  @elleobrien  I have read through everything now.  Only found these typos.  I checked all the links too.  All good.  Just need Ivan to make sure technically correct.

File: content/blog/2021-02-26-february-21-community-gems.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  Plus a new CML feature to launch cloud compute with 
+  Terraform!
+picture: 2021-02-26/feb-gems-cover.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/february-21-community-gems/686
+tags:
+  - Discord
+  - Gems
+  - CML
+  - config
```
Comment (shcheklein) [inline content/blog/2021-02-26-february-21-community-gems.md:23]: config -> Config

what does this tag mean btw? :)

**Comment by shcheklein** [comment]
Looks like image size is a bit smaller than it should it be? it is usually something like `1700x...`

File: content/blog/2021-02-26-february-21-community-gems.md (hunk 1 above)
Comment (elleobrien) [reply content/blog/2021-02-26-february-21-community-gems.md:23]: somebody had a question about local config files! maybe config is not the most useful tag, though :)

**Comment by elleobrien** [comment]
> Looks like image size is a bit smaller than it should it be? it is usually something like `1700x...`

it's been 1300 for a while, i believe. but it's no issue to size up a bit if it looks too tiny

**Comment by shcheklein** [comment]
@elleobrien looks like your commits are not properly attributed? 

Screen Shot 2021-02-26 at 3 40 50 PM

File: content/blog/2021-02-26-february-21-community-gems.md (hunk 1 above)
Comment (shcheklein) [reply content/blog/2021-02-26-february-21-community-gems.md:23]: yep, feel a bit too granular

**Comment by elleobrien** [comment]
> @elleobrien looks like your commits are not properly attributed?
> 
> Screen Shot 2021-02-26 at 3 40 50 PM

Yep yep gotta fix that. was playing with gitlab last night and messed with my global config :)

File: content/blog/2021-02-26-february-21-community-gems.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+### [Q: What's the best way to install the new version of DVC in a Conda environment? I'm concerned about the `paramiko` dependency.](https://discord.com/channels/485586884165107732/563406153334128681/669173874247729165)
+
+When you install DVC via `conda`, it will come with dependencies like
+`paramiko`.
+
+The only exception when installing DVC as a Python library is with with `pip`:
+you might want to specify the kind of remote storage you need to make sure all
+dependencies are present (like `boto` for S3). You can run
+`pip install dvc[<option>]`, with supported options like `[s3]`, `[azure]`,
```
Comment (shcheklein) [inline content/blog/2021-02-26-february-21-community-gems.md:58]: "dvc[<option>]" (double quotes, on some systems it is required)

**Review by shcheklein — APPROVED** [review]
looks great, some minor issues to address. Please feel free to merge whenever you feel it is ready

**Pull request merged**